src/univariate/discrete/negativebinomial.jl

```diff
@@ -59,6 +59,19 @@ function Base.convert(::Type{NegativeBinomial{T}}, d::NegativeBinomial) where {T
 end
 Base.convert(::Type{NegativeBinomial{T}}, d::NegativeBinomial{T}) where {T<:Real} = d
 
+
+# # Interconversion
+# function convert(::Type{NegativeBinomial{T}}, d::NegativeBinomial2) where {T<:Real}
+#     NegativeBinomial{T}(T(d.ϕ), T(d.ϕ / (d.μ + d.ϕ)))
+# end
+# function convert(::Type{NegativeBinomial{T}}, d::NegativeBinomial2Log) where {T<:Real}
+#     NegativeBinomial{T}(T(d.ϕ), T(d.ϕ / (exp(d.η) + d.ϕ)))
+# end
+# function convert(::Type{NegativeBinomial{T}}, d::NegativeBinomial3) where {T<:Real}
+#     NegativeBinomial{T}(T(d.α), T(one(T) / (d.β + one(T))))
+# end
+
+
```

Comment:
Maybe add a comment that conversion methods are defined in src/conversion.jl but I don't think it's helpful to copy code from there in a comment. The implementation might diverge and it creates additional but not really needed changes in this PR.

Reply:
```suggestion
```